cartography/intel/gcp/gke.py

```diff
@@ -35,7 +35,7 @@ def get_gke_clusters(container: Resource, project_id: str, regions: list, common
         res = req.execute()
         data = []
         for item in res.get('clusters', []):
-            item['id'] = f"projects/{project_id}/clusters/{item['name']}"
+            item['id'] = f"projects/{project_id}/location / {item['location']}/clusters/{item['name']}"
```

Comment:
there's an additional space and also we should not pick location, it should be region